openpower/package/hostboot/hostboot.mk

```diff
@@ -16,7 +18,24 @@ HOSTBOOT_INSTALL_TARGET = NO
 HOSTBOOT_ENV_VARS=$(TARGET_MAKE_ENV) \
     CONFIG_FILE=$(BR2_EXTERNAL)/configs/hostboot/$(BR2_HOSTBOOT_CONFIG_FILE) \
     OPENPOWER_BUILD=1 CROSS_PREFIX=$(TARGET_CROSS) HOST_PREFIX="" HOST_BINUTILS_DIR=$(HOST_BINUTILS_DIR) \
-    HOSTBOOT_VERSION=`cat $(HOSTBOOT_VERSION_FILE)`
+    HOSTBOOT_VERSION=`cat $(HOSTBOOT_VERSION_FILE)` 
+
+define HOSTBOOT_APPLY_PATCHES
+       if [ "$(BR2_OPENPOWER_POWER9)" == "y" ]; then \
```

Comment:
Buildroot searches the openpower/package/hostboot and openpower/package/hostboot-$(PKG)_VERSION) for patches.

This worked when he had one release, but adding the p9 code we need to be able to apply specific patches to specific releases. 

Having separate branches that are patch specific is also a solution - but not sure if that would work in a build environment where we build team is releasing drivers daily with different patches.